src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2345,4 +2345,13 @@ Uninstall the following packages: {0}?</value>
     <value>Requires relaunch of Dynamo</value>
     <comment>Preferences | Features | Python | Requires relaunch of Dynamo</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="MessagePackageDepsInStdLib" xml:space="preserve">
+    <value>{1} has dependencies that conflict with the following Stadard Library packages : {2}. {0} cannot override packages in the Standard Library location.</value>
+  </data>
+  <data name="MessageSamePackageInStdLib" xml:space="preserve">
+    <value>The package {1} is already installed as part of the Standard Library. {0} cannot override packages in the Standard Library location.</value>
+  </data>
+  <data name="MessageSamePackageDiffVersInStdLib" xml:space="preserve">
+    <value>A different version of the package {1} is already installed as part of the Standard Library. {0} cannot override packages in the Standard Library location.</value>
+  </data>
```

Comment:
{0}  = "Dynamo"
{1} = The desired package name, ex: MeshToolkit
{2} = List of package names and versions are in conflict.

Reply:
Does 0 come from branding resource provider or something like that?

Reply:
string at position 0 is the application name( Ex. Dynamo, Dynamo for Revit)

Reply:
Yes, I know that. Never mind my question.